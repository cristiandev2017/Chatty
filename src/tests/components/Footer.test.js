import { shallow } from "enzyme";
import Footer from "../../components/Footer";
import React from "react";

describe("Pruebas del componente <Footer/>", () => {
  //Arrange
  const wrapper = shallow(<Footer />);

  test("Comprobar que corra el componente", () => {
    //Como es solo para revisar que funcione y el componente no tiene acciones
    //El Assert sera un snapshot
    expect(wrapper).toMatchSnapshot();
  });

  //Puedo hacer pruebas de elementos
  test("Debe de mostrar el valor del texto", () => {
    //Arrange
    const texto = "&copy; Chatty 2021.";
    //Act
    const textoFooter = wrapper.find("p").text();
    console.log(textoFooter);
  });
});
