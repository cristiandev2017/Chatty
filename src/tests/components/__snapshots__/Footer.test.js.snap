// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas del componente <Footer/> Comprobar que corra el componente 1`] = `
<footer
  className="pt-5"
>
  <div
    className="container text-center"
  >
    <p>
      Chatty 2021.
    </p>
  </div>
</footer>
`;
